packages/terra-props-table/src/PropsTable.jsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import React, { PropTypes } from 'react';
 import { parse } from 'react-docgen';
```

Comment:
`react-docgen` is a dependency for this project. In `packages/terra-props-table`, react-docgen needs to be a dependency.